NEWS.md

```diff
@@ -1,3 +1,7 @@
+# orderly 1.2.22
```

Comment:
Looks like the version number needs to be bumped again

Reply:
Thanks, fixed